style/style.css

```diff
@@ -7,308 +7,519 @@ body {
+  border-radius: 8px;
+  overflow: auto;
+}
+
+.project-container img {
+  overflow: auto;
+  max-width: 100%;
+}
+
+.desc {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.desc h3 {
+  padding: 15px;
+  color: #3a4a42;
+  font-size: 24px;
+  font-weight: 600;
+  text-align: center;
+}
+
+.desc ul {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.desc button {
+  padding: 12px;
+  margin: 20px;
+  color: white;
+  font-weight: 500;
+  font-size: 17px;
+  text-align: center;
+  background: #36b37f;
+  border: #36b37f;
+  border-radius: 4px;
+}
+
+.main3 {
+  height: 47vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-bottom: 40px;
+  margin-top: 40px;
+  background:
+    url("/image/mei.png") bottom left no-repeat,
+    url("/image/duck.png") top right no-repeat;
+}
+
+.main3 h1 {
+  text-align: center;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.main3 p {
+  margin: 20px;
+  font-weight: 400;
+  font-size: 19px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+}
+
+.main3 button {
+  padding: 12px;
+  margin: 20px;
+  color: white;
+  font-weight: 500;
+  font-size: 17px;
+  text-align: center;
+  background: #36b37f;
+  border: #36b37f;
+  border-radius: 4px;
+}
+
+.card-info {
+  display: grid;
+  grid-row-gap: 20px;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.card img {
+  margin: 20px 1px;
+  padding: 20px;
+}
+
+.card h1 {
+  font-weight: 600;
+  font-size: 24px;
+  color: #344563;
+  margin: 20px 1px;
+}
+
+.card ul {
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  margin-left: 15px;
+  margin-right: 15px;
+  margin-bottom: 40px;
+}
+
+.main4 {
+  margin: 24px;
+  margin-top: 60px;
+}
+
+.main4 h1 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+}
+
+.contact-form {
+  margin-top: 58px;
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-form input {
+  margin: 10px 0;
+  padding: 15px 16px;
+  border: 1px solid #d0d9d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.contact-form textarea {
+  font-family: "Inter", sans-serif;
+  height: 85px;
+  padding: 12px 10px;
+  margin: 10px 0;
+  margin-bottom: 24px;
+  border: 1px solid #60c095;
+}
+
+.contact-form button {
+  font-family: "Inter", sans-serif;
+  background: #36b37f;
+  border: #36b37f;
+  width: 130px;
+  color: white;
+  font-size: 17px;
+  font-weight: 500;
+  padding: 12px;
+  align-self: center;
+  border-radius: 4px;
+}
+
+.sc-desktop {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
+  .desktop-intro {
+    background-image: url("/image/header-bg-desktop.png");
```

Comment:
- [ ] Great work adding the desktop background to your project, but unfortunately it's not showing up in the live server as you can see in the screenshots below, Kindly, fix this bug 🙏🏼☺️.
Maybe you miss just a dot `.` before the `url` link `background-image: url("./image/header-bg-desktop.png");` what do you think

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/149575726-e35a271e-db6b-4916-a619-ad8e935781d0.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/149575729-08e0b31b-0d3e-485d-8f0b-3ecf68550204.PNG)

Reply:
### Fixed🚀
<img width="1440" alt="desk" src="https://user-images.githubusercontent.com/40362530/149578207-c6b1d5a2-b6aa-41fa-b0f9-a72ddf179b77.png">